### DISCUSSION TITLE
Invalid Character 'e' Error in ASNmap with AS5650

### DISCUSSION THREAD

**Discussion by byteshiva** [projectdiscovery/asnmap#260] [Q&A] [answered]
**Description:**
When using the ASNmap tool with the command `asnmap -a AS5650`, an error occurs with the following details:

```
$ asnmap -a AS5650
   ___   _____  __              
  / _ | / __/ |/ /_ _  ___ ____ 
 / __ |_\ \/    /  ' \/ _  / _ \
/_/ |_/___/_/|_/_/_/_/\_,_/ .__/
                         /_/ 

		projectdiscovery.io

[INF] Current asnmap version v1.0.6 (latest)
[FTL] invalid character 'e' looking for beginning of value
```

**Steps to Reproduce:**
1. Install the latest version of ASNmap (v1.0.6).
2. Run the command `asnmap -a AS5650` in the terminal.
3. Observe the error message with the "invalid character 'e' looking for beginning of value" issue.

**Expected Behavior:**
The ASNmap tool should execute the command without any errors and provide the expected output for the specified Autonomous System Number (AS5650).

**Additional Information:**
- Operating System: nixos
- ASNmap Version: v1.0.6 (latest)

This issue is affecting the proper functionality of ASNmap, and it would be great if it could be addressed in the next release.
```[tasklist]
### Tasks
- [x] nix-shell -I nixpkgs=https://github.com/NixOS/nixpkgs/archive/nixos-unstable.tar.gz -p asnmap jq
```

**Comment by Mzack9999** [comment, marked as answer]
@byteshiva This is due to asnmap server heavy load. We are probably going to implement rate limits server side that should reduce the pressure on the server shortly.
